src/lamplib/src/genny/tasks/auto_tasks_all.py

```diff
@@ -11,13 +11,17 @@
     Repo,
     ConfigWriter,
     YamlReader,
+    VariantTask,
 )
 
 SLOG = structlog.get_logger(__name__)
 
-def get_all_builds(global_expansions: dict[str, str], project_file_path: str) -> List[CurrentBuildInfo]:
+
+def get_all_builds(
+    global_expansions: dict[str, str], project_file_path: str
+) -> List[CurrentBuildInfo]:
     """
-    Attempts to compute the expansions 
+    Attempts to compute the expansions
```

Comment:
It seems your formatter disagrees with some of the existing formatting, which results in formatting-only changes like this that make the PR harder to parse.

My preference here would be that we either:
a) apply no formatter at all
b) set up an agreed-upon formatter as part of the CI tests (not as part of this ticket)

Reply:
I attempted to what was set up as part of the repo, and explicitly ran `run-genny lint-python --fix`, which _should_ match what we do in CI. I'm a bit mystified by the fact that it fixed anything, but I figured I'd add it.

Reply:
That is, we're supposed to be doing 2 above but it seems like something is wonky.